android/app/src/main/java/host/exp/exponent/MainApplication.java

```diff
@@ -6,8 +6,31 @@
 import java.util.Arrays;
 import java.util.List;
 
+import expo.core.interfaces.Package;
+import expo.loaders.provider.interfaces.AppLoaderPackagesProviderInterface;
+// import expo.modules.ads.admob.AdMobPackage;
```

Comment:
Are we just keeping these commented out bits here for future reference?  Same with the package list below?

Reply:
Yes, exactly. These were added by default in Expo 32. Thought it would be useful to comment out instead of remove them. Probably fine to just remove them, though, so as not to cruft the code. Will do.